### ISSUE TITLE
app crashes immediately after starting

### ISSUE THREAD

**Issue by ed-jpa** [gamesir123/app#95]
App instantly crashes on redmi note 10 pro. Both the version on playstore and the most up to date version on github crash immediately.

**Comment by Jballer1** [comment]
Hello, I've been experiencing the same issue intried deleting cache data and Uninstaller. The only thing that seemed to work was installing a previous version of the app from the website. I'm on android...

**Comment by ed-jpa** [comment]
Right, app version 4.2.0 starts, but then the remapping function doesn't seem to work. I can configure the buttons but the screen doesn't recognize touches. Does it work for you?

**Comment by Jballer1** [comment]
I installed 4.1.1 I can remap but as soon as I change mode the app crashes but I can continue to use my setup. When using the gamepad x2 usb with the Xbox game pass it works perfect.
